sru/custom_m_utils.py

```diff
@@ -0,0 +1,188 @@
+from torch import nn
```

Comment:
Can we rename the file to sth like `utils.py`? `custom_m` is not informative and doesnot sound like a great name to me.